app/src/main/java/com/bilginyuksel/sdk/push/utils/Util.java

```diff
@@ -0,0 +1,21 @@
+package com.bilginyuksel.sdk.push.utils;
+
+import android.app.PendingIntent;
+import android.content.Context;
+import android.content.Intent;
+import android.os.Build;
+
+import com.bilginyuksel.sdk.push.MainActivity;
+
+public class Util {
```

Comment:
Unnamed util class does not imply anything. I think we should give a meaningful name to it. And also we shouldn't put `createPendingIntent` method here.